code/game/gamemodes/cult/ritual.dm

```diff
@@ -169,7 +169,7 @@ var/engwords = list("travel", "blood", "join", "hell", "destroy", "technology",
 	if(!iscultist(user))
 		user << "<span class='notice'>You can't mouth the arcane scratchings without fumbling over them.</span>"
 		return
-	if(user.is_muzzled())
+	if(user.can_speak())

```

Comment:
does can_speak() return true for not being able to speak? or have you just inverted the logic?